src/libs/sfdk/dockervirtualmachine.cpp

```diff
@@ -83,12 +84,20 @@ QString dockerPath()
     if (!path.isNull())
         return path;
 
+    QString dockerBinary = DOCKER;
+    QString pathVariable = "PATH";
+    QChar pathSeparator = ':';
+    if (Utils::HostOsInfo::isWindowsHost()) {
+        dockerBinary.append(".exe");
+        pathVariable = "Path";
```

Comment:
I think this is not needed. The docs for QProcessEnvironment says "On Windows, the variable names are case-insensitive, but case-preserving. QProcessEnvironment behaves accordingly." and looging at QProcessEnvKey in qproess_p.h it really seems to be so.